docs/release-notes/v11.2.0.md

```diff
@@ -0,0 +1,103 @@
+---
+id: v11.2.0
+title: Version v11.2.0 Release Notes
+sidebar_label: v11.2.0
+image: "img/release-note-link-preview.png"
+---
+
+_June 8th, 2023_
+
+## Console
+
+### New Features
+
+#### Users and Service Accounts portals
+
+Analogously to what we have implemented for groups management, from now on even users and service accounts have proper management portals. These dedicated pages provide specific governance tools based on the nature of the identity, be it a user, a group or a service account.
+
+**Users section** allows Company Owners to view all user accounts which have access to the Company, providing a way to add new ones or to completely remove access to the Company for existing ones.
+For more information, [go to the documentation](/development_suite/identity-and-access-management/manage-users.md).
+
+![Users Portal](./img/overview/users-portal.png)
+
+**Service Accounts section** allows Company Owners to view all service accounts of the Company, with the possibility to create new ones or permanently delete existing ones.
+For more information, [go to the documentation](/development_suite/identity-and-access-management/manage-service-accounts.md).
+
+![Service Account Portal](./img/overview/service-accounts-portal.png)
```

Comment:
the screenshot should be replaced: it shows sections not released yet

Reply:
solved